package.json

```diff
@@ -34,6 +34,7 @@
     "css-loader": "^0.26.2",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.10.1",
+    "fork-ts-checker-webpack-plugin": "^0.2.7",
```

Comment:
Ah, this makes sense. Congrats! You were the first to hit this issue :). Basically, it's quite often for Encore to have optional features. Think about `sass-loader` - we don't have that in our `dependencies` because we force the user to add this manually. We add it in `devDependencies` just so we can use it in our tests.

But `fork-ts-checker-webpack-plugin` is different, because (unlike all the optional loaders), we explicitly *require* this module (for loaders, you just mention the loader name and webpack requires it later - so the linter doesn't pick up on this).

tl;dr; The linter is correct in a sense: Encore is using this package in its actual *code*, not just in its tests. But it's wrong because we're manually checking to make sure this package exists before using it.

So, add `// eslint-disable-line` after the `require` line in `forked-ts-types.js` to silence this lint problem. THEN put it in `devDependencies` :)